src/app/components/home/add-marios/add-marios.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="container">
+  <div class="back-button-container">
+    <a [routerLink]="['/home']">
```

Comment:
You could also add to button (click) and on click call `this.router.navigateByUrl('/home');`